.pre-commit-config.yaml

```diff
@@ -20,6 +21,7 @@ repos:
     rev: 6.0.0
     hooks:
       - id: flake8
+        additional_dependencies: [Flake8-pyproject]
```

Comment:
I wonder if we should swap to ruff instead https://github.com/charliermarsh/ruff ?

I have been using it in other projects and I definitely appreciate the speed improvements (plus it is basically on par with flake8)

Reply:
I myself prefer ruff as well. What about dealing with it in a separated PR (there will be slight linting changes with flake8-isort) ?